eZ/Publish/API/Repository/Tests/LocationServiceAuthorizationTest.php

```diff
@@ -115,6 +115,35 @@ public function testLoadLocationByRemoteIdThrowsUnauthorizedException()
     }
 
     /**
+     * Test for the loadLocations() method.
+     *
+     * @see \eZ\Publish\API\Repository\LocationService::loadLocations()
+     * @depends \eZ\Publish\API\Repository\Tests\LocationServiceTest::testLoadLocations
```

Comment:
Bad news, this test gets skipped ;) PHPUnit needs FQCN without leading `\`, so:
`@depends eZ\Publish\API\Repository\Tests\LocationServiceTest::testLoadLocations`

*Of course it breaks the ability to navigate to the method in PhpStorm. `@depends` is really broken...*